src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -1120,11 +1255,13 @@ await DownloadFileAsync(file.FilePath, destination, cancellationToken)
             string caption,
             InlineKeyboardMarkup replyMarkup = default,
             CancellationToken cancellationToken = default,
-            ParseMode parseMode = default
+            ParseMode parseMode = default,
+            IEnumerable<MessageEntity> captionEntities = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.